kotlin-multiplatform/composeApp/build.gradle.kts

```
@@ -2,17 +2,45 @@ import org.jetbrains.compose.desktop.application.dsl.TargetFormat
 import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
 import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 import org.jetbrains.compose.ExperimentalComposeLibrary
+import java.util.Properties
+import java.io.FileInputStream
 
 plugins {
     alias(libs.plugins.kotlinMultiplatform)
     alias(libs.plugins.androidApplication)
     alias(libs.plugins.composeMultiplatform)
     alias(libs.plugins.composeCompiler)
     alias(libs.plugins.detekt)
+    alias(libs.plugins.buildkonfig)
 
     id("quickstart-conventions")
 }
 
+fun loadEnvProperties(): Properties {
+    val properties = Properties()
+    val envFile = rootProject.file("../.env")
+
+    if (envFile.exists()) {
+        FileInputStream(envFile).use { properties.load(it) }
+    } else {
+        val requiredEnvVars = listOf(
+            "DITTO_APP_ID",
+            "DITTO_PLAYGROUND_TOKEN",
+            "DITTO_AUTH_URL",
+            "DITTO_WEBSOCKET_URL"
+        )
+
+        for (envVar in requiredEnvVars) {
+            val value = System.getenv(envVar)
+                ?: throw RuntimeException("Required environment variable $envVar not found. Please create a .env file or set environment variables.")
+            properties[envVar] = value
+        }
+    }
+    return properties
+}
```

Maybe worth also validating that `.env` has the values we're looking for, in case we add or rename something in the future and somebody still has their old file

```suggestion
fun loadEnvProperties(): Properties {
    val properties = Properties()
    val requiredEnvVars = listOf(
        "DITTO_APP_ID",
        "DITTO_PLAYGROUND_TOKEN",
        "DITTO_AUTH_URL",
        "DITTO_WEBSOCKET_URL"
    )
    val envFile = rootProject.file("../.env")

    if (envFile.exists()) {
        FileInputStream(envFile).use { properties.load(it) }
    } else {
        for (envVar in requiredEnvVars) {
            val value = System.getenv(envVar)
            properties[envVar] = value
        }
    }

    for (envVar in requiredEnvVars) {
        properties.getProperty(envVar)
            ?: throw RuntimeException("Required environment variable $envVar not found. Please create a .env file or set environment variables.")
    }

    return properties
}
```